source/learn/getting-started/auth-with-rom.html.md

```diff
@@ -0,0 +1,257 @@
+---
+chapter: Getting Started
+title: Simple authentication with ROM and Rails
+---
+
+## Initial setup
+
+This guide describes how to add a simplest form of user authentication using ROM, Warden and Rails.
+
+> Please note that this guide is only for illustrative purposes and does not include validations 
+
+First of all, let's add following gems to the Gemfile:
+
+```ruby
+# Gemfile
+gem 'rom-rails'
+gem 'rom-sql'
+gem 'rom-repository'
+gem 'rom'
+gem 'warden'
+gem 'bcrypt'
+gem 'sqlite3'
+```
+
+Now we will need to add configuration for ROM to work. In this example we are going to use SQLite3 to store our data:
+
+```ruby
+# config/initializers/rom.rb
+ROM::Rails::Railtie.configure do |config|
+  config.gateways[:default] = [:sql, "sqlite://db/dev.db"]
+end
+```
+After this, we will need to add ROM rake tasks in our Procfile
+
+```ruby
+# Procfile
+require 'rom/sql/rake_task'
+```
+
+When this is done, let's generate a migration that adds users table to our database:
+
+```bash
+rake db:create_migration[create_users]
+# <= migration file created db/migrate/20161109173831_create_users.rb
+```
+
+Let's open up our migration file and add the following lines:
```

Comment:
you gotta add an empty line here, for some reason middleman doesn't render html correctly without it